gravitee-am-reporter/gravitee-am-reporter-mongodb/src/main/java/io/gravitee/am/reporter/mongodb/audit/constants/MongoAuditReporterConstants.java

```diff
@@ -38,6 +38,7 @@ private MongoAuditReporterConstants(){}
   public static final String FIELD_ACCESS_POINT_ID = "accessPoint.id";
   public static final String INDEX_REFERENCE_TIMESTAMP_NAME = "r1t_1";
   public static final String INDEX_REFERENCE_TYPE_TIMESTAMP_NAME = "r1ty1t_1";
+  public static final String INDEX_REFERENCE_TYPE_STATUS_SUCCESS_TIMESTAMP_NAME = "r1ty1success1t_1";
```

Comment:
Is the index name short enough ?
Thinking of DocumentDB

Reply:
I shorted it to r1ty1s1t_1
same length like existing INDEX_REFERENCE_ACTOR_ID_TARGET_ID_TIMESTAMP_NAME (10)